static_report/src/components/shared/Tables/TableList/CRTableListItem/CRTableListAssertions.tsx

```diff
@@ -249,45 +238,61 @@ export function CRTargetTableAssertionsSummary({
   }
 
   return (
-    <Flex gap={1} alignItems="center">
-      <Flex
-        alignItems="center"
-        borderRadius="md"
-        bgColor={isPassed ? '#DEFFEB' : '#FFE8F0'}
-        color={isPassed ? '#1F7600' : '#F60059'}
-        py={0.5}
-        px={1.5}
-        gap={1}
-      >
+    <ColumnAssertionLabelWidget
+      isPassed={isPassed}
+      total={total}
+      failed={failed}
+      isComparison
+      comparisonDelta={<ComparisonDelta delta={delta} total={total} />}
+      icon={
+        <ComparisonLabelIcon
+          isPassed={isPassed}
+          isFailedEqual={isFailedEqual}
+          isMoreFailed={isMoreFailed}
+        />
+      }
+    />
+  );
+}
+
+function ComparisonLabelIcon({
```

Comment:
Would this be reusable from `components/Assertions/..` Just did a shallow search and found these

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/11186480/188780064-db38f37b-3ba8-4b47-b11d-9d09c851d4c3.png">
<img width="509" alt="image" src="https://user-images.githubusercontent.com/11186480/188780130-abb3a908-39df-4d4c-b385-ef2b3ef77dfc.png">
<img width="517" alt="image" src="https://user-images.githubusercontent.com/11186480/188780177-deb50d6f-443a-464c-8993-0d3d250b782f.png">